### PR TITLE
ASC to CAAS

### LINKED ISSUE #3010: Change ASC to CAAS
ASC is now known as CAAS according to this [news article](https://wp.stolaf.edu/news/padula-family-supports-new-center-for-advising-and-academic-support. ). This means we need to change:
- [x] Dictionary term and definition?
  - the new one is [pretty sparse](https://wp.stolaf.edu/academic-support/) comparatively
  - but we could detail the "portfolio of services" provided in the webpage sidebar
- [x] Building hours names and times
  - source updated [here](https://wp.stolaf.edu/academic-support/asc-hours/)

### PR DESCRIPTION
**Pull request by drewvolz**
Resolves #3010

**Dictionary**
* Update CAAS dictionary definition
* Update references to CAAS in the definitions for SAS and DAC

**Hours**
* Update Saturday opening time to 10am (from 11:30am) for writing desk
* Add hours for the CAAS study space
  * The area is available for individual study, group study, and tutoring sessions

<details>
<summary>📸 Screenshots</summary>

~ | ~
--|--
Screen Shot 2019-04-09 at 1 08 34 PM | Screen Shot 2019-04-09 at 1 08 41 PM
Screen Shot 2019-04-09 at 1 08 54 PM | Screen Shot 2019-04-09 at 1 08 56 PM
Screen Shot 2019-04-09 at 1 09 04 PM | Screen Shot 2019-04-09 at 1 11 03 PM
Screen Shot 2019-04-09 at 1 11 58 PM | Screen Shot 2019-04-09 at 1 12 02 PM
Screen Shot 2019-04-09 at 1 12 30 PM | Screen Shot 2019-04-09 at 1 10 52 PM

</details>

### CODE REVIEW

**Comment by hawkrives** [comment]
Could you add the Fireside writing desk location as well, please?

**Review by rye — DISMISSED** [review]
Approved. (@hawkrives' suggestion can go in this PR or a separate one, I don't really care.)

**Comment by drewvolz** [comment]
@hawkrives you sent me an image of the fireside desk writing hours, but I cannot confirm those online. Are we wanting to go with the hours from your photo?

**Comment by hawkrives** [comment]
Yes, please

**Comment by drewvolz** [comment]
Going with the posted hours [here](https://wp.stolaf.edu/academic-support/writing-help/) for 3 of the 4 locations, and the photo @hawkrives provided for the fireside hours.

~ | ~
--|--
Screen Shot 2019-04-19 at 11 19 12 AM | Screen Shot 2019-04-19 at 11 19 16 AM
Screen Shot 2019-04-19 at 11 20 39 AM

**Comment by drewvolz** [comment]
@hawkrives I've now separated out the 4 writing desk locations and their hours

File: data/building-hours/5-4-caas-writing-desk.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: CAAS – Writing Desk
+category: Help and Support
+
+schedule:
+  - title: Rolvaag 354
```
Comment (hawkrives) [inline data/building-hours/5-4-caas-writing-desk.yaml]:
```suggestion
  - title: Rølvaag 354
```

File: data/building-hours/5-4-caas-writing-desk.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - title: Reference Room
+    hours:
+      - {days: [Mo, Tu, We, Th, Fr], from: '9:00am', to: '5:00pm'}
+      - {days: [Mo, Tu, We, Th], from: '7:00pm', to: '10:00pm'}
+      - {days: [Sa], from: '11:00am', to: '5:00pm'}
+      - {days: [Su], from: '12:00pm', to: '5:00pm'}
+      - {days: [Su], from: '7:00pm', to: '10:00pm'}
+    notes: Times by appointment.
+
+  - title: CHM library
```
Comment (hawkrives) [inline data/building-hours/5-4-caas-writing-desk.yaml]:
```suggestion
  - title: CHM Music Library
```

Comment (hawkrives) [reply data/building-hours/5-4-caas-writing-desk.yaml]: and _maybe_ even drop the "CHM" prefix?

**Review by hawkrives — DISMISSED** [review]
some naming tweaks

**Comment by hawkrives** [comment]
@drewvolz Could you upload another screenshot of the hours list view for Writing Desk please? I want to make sure they're roughly the same length.

**Comment by drewvolz** [comment]
@hawkrives 
image

**Comment by drewvolz** [comment]
I'll drop "CHM" ...edit: I like it.

image

**Review by hawkrives — APPROVED** [review]

**Pull request merged**